lib/display/display.ts

```diff
@@ -96,6 +96,12 @@ export function displayDependencies(
 
     result.push(`\n${leftPad(dependencyName, 2)}`);
 
+    if (fileSignaturesDetails && fileSignaturesDetails[dependencyId]?.purl) {
+      result.push(
+        leftPad(`purl: ${fileSignaturesDetails[dependencyId].purl}`, 2),
+      );
+    }
```

Comment:
i would log a debug message for the else code path , in case the purl could miss